EVM and High Level Languages.md

```diff
@@ -0,0 +1,95 @@
+# The EVM and High-Level Smart Contract Programming Languages
+
+## Introduction
+
+Bitcoin has always been the highest volume and value cryptocurrency in digital money. It runs on blockchain technology. Blockchain is a peer to peer, public, encrypted, trust-less ledger. Script is the name of the scripting language that constructs Bitcoin. Script is a mid to lower capability programming language. Public and private key transacting is the model for signing transactions. Besides transacting digital assets, there is no extra utility built into Bitcoin. 
+
+Ethereum Blockchain technology offers limitless use case scenarios. A virtual computer deployed on the Ethereum Blockchain creates many possible use cases. Smart contracts are programs written in a high-level programming language. Think of JavaScript syntax with a name of Solidity or Vyper.  These languages build Smart contract code that compiles for use on the Ethereum Blockchain. This computing system revolutionizes privacy, security, and freedom otherwise unrealized in present-day governance.
+
+## Ethereum Virtual Machine Feature Overview
+
+High-level is a vague description of a computing platform's capability. Precise performance metrics would go beyond the scope of this article. The Ethereum Blockchain is a powerful and isolated virtual computing environment. Computing features of the Ethereum Virtual Machine (EVM) include: 
+
+* Predefined function libraries 
+* Function constructors and destructors
+* Function overrides 
+* High-level logic
+* Looping and recursion 
+* Full range of expressions for logical and mathematical arguments
+* Control structures
+* More than one variable can pass in and out of a function 
+* User ability to define custom functions 
+* Inheritance
+* Polymorphism 
+* Sand boxing 
+* Two-way interaction with other smart contracts 
+
+Simple transaction protocol initiates the deployment of very powerful applications on the EVM. The Ethereum protocol is trust-less; secured by peer-to-peer consensus and data encryption. It is very difficult, impossible, to access unauthorized information. The program code is verifiable and impossible to alter. Always available to every user as the same version and build. Pushing updates that users ignore can become a thing of the past.
```

Comment:
A lot of this paragraph is a bit basic, and a bit vague. Again, this is a post about an EVM deep dive. A couple of words here and there about what these terms mean is all that's needed.